distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1543,6 +1543,196 @@ public void testFederatedDownloadProductToCacheOnlyCacheDisabled() throws Except
                 .resolve(CSW_SOURCE_ID + "-" + metacardId + ".ser")), is(false));
     }
 
+    @Test
+    public void testTwoUsersSameProductRetrySuccess() throws Exception {
+
+        CometDClient cometDClient1 = setupCometDClient(Arrays.asList(NOTIFICATIONS_CHANNEL,
+                ACTIVITIES_CHANNEL));
+        CometDClient cometDClient2 = setupCometDClient(Arrays.asList(NOTIFICATIONS_CHANNEL,
+                ACTIVITIES_CHANNEL));
+
+        String filename = "product1.txt";
+        String metacardId = generateUniqueMetacardId();
+        String resourceData = getResourceData(metacardId);
+
+        setupCswServer(metacardId, filename, resourceData);
+        setupCswServer(metacardId, filename, resourceData);
+
+        String restUrl = REST_PATH.getUrl() + "sources/" + CSW_STUB_SOURCE_ID + "/" + metacardId
+                + "?transform=resource" + "&session=" + cometDClient1.getClientId();
+
+        // Verify that the testData from the csw stub server is returned.
+        // @formatter:off
```

Comment:
I think the reason the formatter was turned off here (and I’ve noticed the formatter is turned off for lines like this in other TestFederation tests as well), is that when formatted, this line puts every dot on a new line, resulting in the whole being harder to read. When it is all on one line it almost reads like a sentence.